features/legacy/frontend/cart_promotions_percentage.feature

```diff
@@ -69,5 +69,5 @@ Feature: Checkout percentage discount promotions
         And I added product "Buzz" to cart, with quantity "1"
         When I add product "Woody" to cart, with quantity "3"
         Then I should still be on the cart summary page
-        And "Promotion total: -€586.25" should appear on the page
-        And "Grand total: €1,088.75" should appear on the page
+        And "Promotion total: -€544.38" should appear on the page
+        And "Grand total: €1,130.62" should appear on the page
```

Comment:
Well, in fact it does not :smile: It's a funny thing:
- we have order containing 4x **Potato** (€200.00 each), 1x **Buzz** (€500.00) and 3x **Woody** (€125.00)
- it gives us total of €1675.00
- promotion **PR1** gives 25% off on order (we have 8 units, 5 required) -> €1675.00 - 25% = €1256.25
- promotion **PR2** gives 10% off on order (as its total is above €300.00) -> €1256.25 - 10% = €1130.62

so as we see, order is calculated correctly :D I have no idea how it worked before, by for me it's good now.